Flutter/DateRangePicker/date-navigations.md

```diff
@@ -139,7 +139,7 @@ class _MyAppState extends State<MyApp> {
 Initially or during the run time, you can selects the multiple dates programmatically by using the [selectedDates](https://pub.dev/documentation/syncfusion_flutter_datepicker/latest/datepicker/DateRangePickerController/selectedDates.html) of `DateRangePickerController` property. It is only applicable when the `selectionMode` is set to `DateRangePickerSelectionMode.multiple`.
 
 {% tabs %}
-{% highlight dart hl_lines="5 9 24 25" %}
+{% highlight dart hl_lines="6 11 12 13 14 15 16 26 27" %}
```

Comment:
Check the line numbers

6, 10, 11, 12, 13, 14, 15, 25, 26

Reply:
Check the line numbers

6, 10, 11, 12, 13, 14, 15, 25, 26

Reply:
25th line highlighted.